assets/js/icons/icon/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { cloneElement, isValidElement } from 'wordpress-element';
+import { cloneElement, isValidElement } from '@wordpress/element';
```

Comment:
Huh, odd - why did we ever directly import this in the first place?

Reply:
No idea. I checked the changelogs to ensure it was available on supported versions.